simulators/totalpower/__init__.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, channels=14):
         self.externalNoise = 0
         self.toggle = 0
 
+        self.diagnostic_string = ''
```

Comment:
Saving this string as a class member seems unnecessary, the C command only serve diagnostic purposes